### DISCUSSION TITLE
Is there any chance that this database will not remain opensource in near future?

### DISCUSSION THREAD

**Discussion by Nadir-Pervez** [apache/cloudberry#458] [Q&A] [answered]
Is there any chance that this database will not remain opensource in near future?

**Comment by tuhaihe** [comment, marked as answer]
No. It will be open source always. Our goal is to make Cloudberry the best open-source alternative to Greenplum.

We want to make the Cloudberry Database neutral and community-driven, as we recognize that if Cloudberry is owned by one company, users and developers will still face an unpredictable future. Now we're talking with Apache Software Foundation and Linux Foundation on hosting Cloudberry with them.
